src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -311,8 +311,8 @@ Start, Stop, Restart
 
          cylc get-resources api-key
 
-      Add the following lines to the bottom of the :cylc:conf:`flow.cylc` file replacing
-      ``xxx...`` with your API key:
+      Add the following lines to the ``[runtime]`` section of the
+      :cylc:conf:`flow.cylc` file replacing
```

Comment:
What was wrong with this instruction?

> flow.cylc file replacing ...

... replacing what???

Reply:
As Joe observed we are modifying existing namespaces in the runtime section, not adding extra to the bottom of the config.

Reply:
Hmm, I'm sure the way it was written was correct, has the tutorial changed since Cylc 7?

Reply:
It was correct, but this is more correct.

Reply:
> has the tutorial changed since Cylc 7 like the other example in cylc-flow?

It has!

During the transfer of the tutorials some blank task sections appear to have been added:

https://github.com/metomi/rose/blob/2019.01.x/etc/tutorial/runtime-tutorial/suite.rc
https://github.com/cylc/cylc-flow/blob/master/cylc/flow/etc/tutorial/runtime-tutorial/flow.cylc

Reply:
OK, will change on the docs side